app/assets/stylesheets/styles.scss

```diff
@@ -1,14 +1,14 @@
-$orange: #D45E00;
+$orange: #d45e00;
 $blue: #00bcd4;
-$red: #D4002E;
+$red: #d4002e;
 
 
 .top-bar {
-  height: 50px;
   background-color: #ffffff;
-  font-family: Roboto, sans-serif;
-  box-shadow: 0 1px 6px rgba(0, 0, 0, 0.12), 0 1px 4px rgba(0, 0, 0, 0.24);
   border-radius: 2px;
+  box-shadow: 0 1px 6px rgba(0, 0, 0, 0.12), 0 1px 4px rgba(0, 0, 0, 0.24);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.12)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `rgba(0, 0, 0, 0.24)` should only be used in variable declarations; they should be referred to via variable everywhere else.